main.go

```diff
@@ -40,109 +48,37 @@ func main() {
 		dbPassword = "@"
 	}
 
-	var err error
+	// var err error
```

Comment:
Remove commented out code

Reply:
Removed this comment. However, I am leaving other commented out code as there seems to be some valuable information for future development that would be more easily accessible if left inside.